packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -55,8 +55,10 @@ export default class ConfigParser {
             /**
              * For Sauce Labs RDC we need to determine if the config file has a `testobject_api_key`
              * If so, we need to provide a boolean to the `detectBackend` to set the correct hostname
+             *
+             * NOTE: This will not work for multi remote
              */
-            const isRDC = this._capabilities.some(capability => 'testobject_api_key' in capability)
+            const isRDC = Array.isArray(this._capabilities) && this._capabilities.some(capability => 'testobject_api_key' in capability)
```

Comment:
please add tests for this

Reply:
Will check it tomorrow, already evening here, unless someone else want's to add tests for this one

Reply:
we don't have a smoke test for multiremote (these are tests within the `/tests` directory). For the scope of this PR it would be sufficient to have a unit tests for this change.

Reply:
@christian-bromann 
I've added 2 tests but I couldn't find a better way then this, please check if you agree